snippets/snippets.code-snippets

```diff
@@ -190,4 +190,44 @@
         ],
         "description": "Define a class which extends another class and implements some classes",
     },
+    "For Loop block": {
+        "prefix": "for",
+        "body": [
+            "for ( var ${1:var_name} = ${2:} ; ${1:} ${3:<} ${4:} ; ${1:}${5:++} ){",
+            "/// Enter For Loop Body",
+            "}"
+        ],
+        "description": "Create a For-Loop Block",
+    },
+    "For In Loop  block": {
+        "prefix": "fin",
+        "body": [
+            "for ( var ${1:var_name} in ${2:} ){",
+            "/// Enter For_In Loop Body",
+            "}"
+        ],
+        "description": "Create a For_In-Loop Block",
+    },
+    "while Loop block": {
+        "prefix": "while",
+        "body": [
+            "var ${1:var_name} = ${2:} ;",
+            "while( ${1:} ${3:<} ${4:} )///condition",
```

Comment:
Removing the **variable declaration** part would be better as different kind of conditions can be there in while loops. It will be annoying to remove the variable code every-time we write loops.